opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -72,8 +95,22 @@ ENV CATALINA_HOME /usr/share/tomcat
 ENV PATH $CATALINA_HOME/bin:$PATH
 WORKDIR $CATALINA_HOME
 
+ARG USER_ID=${TOMCAT_USER_ID:-1000}
+ARG GROUP_ID=${TOMCAT_GROUP_ID:-1000}
+
+# Tomcat user and group created again on actual image definition to run as user.
+RUN groupadd -r tomcat -g ${GROUP_ID}
+RUN useradd -u ${USER_ID} -g tomcat -d ${CATALINA_HOME} -s /sbin/nologin \
+            -c "Tomcat user" tomcat
+
+# Change CATALINA_HOME ownership to tomcat user and tomcat group on actual definition
+RUN chown -R tomcat:tomcat ${CATALINA_HOME}
+RUN chmod -R 400 ${CATALINA_HOME/conf}
```

Comment:
This doesn't really make sense to me, given that the files haven't been copied in yet.